Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: test
 deps:
 	npm install -g eslint mocha istanbul
-	npm install -g eslint-config-airbnb eslint-plugin-jsx-a11y eslint-plugin-import eslint-plugin-react
+	npm install -g eslint-config-airbnb eslint-plugin-jsx-a11y@2.2.3 eslint-plugin-import eslint-plugin-react
```

Comment:
you should remove these because they are in DevDeps and will be installed via npm install